Makefile

```diff
@@ -1,6 +1,6 @@
 .SILENT:
 
-SHELL=/usr/bin/env bash -O globstar
+#SHELL=/usr/bin/env bash -O globstar
```

Comment:
I don't know exactly why this was needed. Probably from another Makefile (we use the same layout a lot) where this happens to be needed. So it's ok to remove it

Reply:
Lol I’ve been commenting and uncommenting this out for months because having this line doesn’t work on my Mac and I guess I forgot to remove it here. But if we remove it then it makes it easy for me :P